src/xdocs/config_filters.xml

```diff
@@ -124,13 +129,6 @@
             SuppressWithPlainTextCommentFilter</a> or similar filter must be used.
           </p>
       </subsection>
-      <subsection name="Notes" id="SuppressionCommentFilter_Notes">
-        <p>
```

Comment:
Same.

Reply:
https://github.com/checkstyle/checkstyle/blob/b7cee495898a0f374ac9c4859f9dd71c0c64f153/src/xdocs/config_filters.xml#L194

Notes now appear after the properties, isn't that the desired case?